src/component/language/language.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { traceEvent } from '../../lib/telemetry.js';
+import './language.scss';
+import Localization from '../../localization/localization';
+
+class Language extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showList: false
+        };
+    }
+
+    onButtonClick = (e) => {
+        this.setState((prev) => {
+            return {
+                showList: !prev.showList
+            }
+        });
+    }
+
+    onLanguageClick = (key, e) => {
+        Localization.getLocalizedString().setLanguage(key);
+        this.setState((prev) => {
+            return {
+                showList: !prev.showList
+            }
+        });
+        e.stopPropagation();
+        this.props.reloadMain();
+    }
+
+    render() {

```

Comment:
After language is changed, will the url changed? Something like; https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-node-node-getstarted for en-us and https://docs.microsoft.com/zh-cn/azure/iot-hub/iot-hub-node-node-getstarted for zh-cn.

This has two benefit:
* Better SEO for certain language
* Good for sharing: e.g. A Chinese could simply share the Chinese content to his friend.